### ISSUE TITLE
Add settings import to command line

### ISSUE THREAD

**Issue by wolf99** [microsoft/terminal#10083]
# Description of the new feature/enhancement

Command line argument allowing the import of new settings files.

This would make scripting of environment setup a bit less fiddly.

Something like

```console
> wt import mySettings.json
Settings imported successfully
```

Potentially could validate settings during import to prevent overwriting existing settings with invalid ones

```console
> wt import myDodgySettings.json
Settings not valid, import failed.
Ln # 8: "HoopyFrood" is not a valid setting
Ln # 9: "KnowsWhereHisTowelIs" is an unknown value for setting "ArthurDent"
    Acceptable values are "Confused", "Bewildered", or "JustWantsANiceCupOfTea".
````

If getting really fancy, the command could update valid but older settings to newer schemas

```console
> wt import myValidButOldSettings.json
Whoa, there Magrathean, those settings look a little stale!
Settings updated to new schema, imported successfully
```

**Comment by DHowett** [comment]
Thanks for the request! We've got a request for supporting import/export for color schemes, and we don't _hate_ the idea . . . but we're wondering what value it adds over "copy the file over the existing settings.json"?

:smile:

**Comment by DHowett** [comment]
(@miniksa suggests that finding the path to the settings file "sucks ass")

**Comment by wolf99** [comment]
I agree with @miniksa 😊, finding the path is at best non-intuitive. 

Additionally the path is fugly, but more importantly it includes your username. For most users this is their actual name. I don't really want that in scripts that are in remote SCM systems or the like. 

Further, using an action rather than a path abstracts the path so that WT can change the underlying path used as it may need, e.g. for preview WT vs main WT vs user installation vs system installation, etc.

I added the extra functionality (validation, etc) in the request just for fun really, but they could be facilitated in such a feature. This would mean that the person doing the configuring (who may not be the same person as the user), can easily know if settings are valid without needing to open the application.

**Comment by KalleOlaviNiemitalo** [comment]
```
> wt import mySettings.json
Settings imported successfully
```

Outputting that to the console of the parent process would have the same difficulties as `wt --help` in <https://github.com/microsoft/terminal/issues/7258>. Just providing an exit code for batch files seems easier, though.

**Comment by wolf99** [comment]
Yeah, exit codes to be preferred. 
The sample outputs I've shown here are more to show different functionality for the purposes if this thread only.

**Comment by zadjii-msft** [comment]
As noted in #17329, we probably want to support YAML from https://www.commands.dev/ as well.
